internal/log_unix.go

```diff
@@ -49,10 +51,19 @@ func SetupHookLogging(hooks []config.LogrusHook, componentName string) {
 		case "syslog":
 			checkSyslogHookParams(hook.Params)
 			setupSyslogHook(hook, level, componentName)
+		case "std":
+			setupStdLogHook(level)
+			stdLogAdded = true
 		default:
 			logrus.Fatalf("Unrecognised logging hook type: %s", hook.Type)
 		}
 	}
+	if !stdLogAdded {
+		logrus.Info("No std logger config found. Enabling at INFO level by default")
```

Comment:
I'm not really sure if we need to log this? It just seems like noise.

Reply:
Yep, I was pretty much just using it for a quick test. Could take it out.